book/svm.qmd

```diff
@@ -207,18 +224,17 @@ Where $\alpha_i$ are again Lagrange multipliers.
 ### Hybrid SSVMs
 
 Finally, @VanBelle2011b noted that the ranking algorithm could be updated to add the constraints of the regression model, thus providing a model that simultaneously optimizes for ranking whilst providing continuous values that can be interpreted as survival time predictions.
-This results in the hybrid SSVM with constraints:
+This results in the hybrid SSVM with constraints $\forall i = 1,...,n$:
 
 $$
 \begin{aligned}
 & \min_{\bb, b, \xi, \xi', \xi^*} \frac{1}{2}\|\bb\|^2 + \textcolor{CornflowerBlue}{\gamma\sum_{i =1}^n \xi_i} + \textcolor{Rhodamine}{\mu \sum^n_{i=1}(\xi_i' + \xi_i^*)} \\
 & \textrm{subject to}
 \begin{dcases}
-\textcolor{CornflowerBlue}{g(\xx_i) - g(\xx_{j(i)})} & \textcolor{CornflowerBlue}{\geq t_i - t_{j(i)} + \xi_i} \\
-\textcolor{Rhodamine}{\delta_i g(\xx_i)} & \textcolor{Rhodamine}{\leq \delta_i t_i + \xi^*_i} \\
+\textcolor{CornflowerBlue}{g(\xx_i) - g(\xx_{j(i)})} & \textcolor{CornflowerBlue}{\geq t_i - t_{j(i)} - \xi_i} \\
+\textcolor{Rhodamine}{g(\xx_i)} & \textcolor{Rhodamine}{\leq t_i + \xi^*_i, i:\delta_i=1} \\
 \textcolor{Rhodamine}{g(\xx_i)} & \textcolor{Rhodamine}{\geq t_i - \xi'_i} \\
 \textcolor{CornflowerBlue}{\xi_i}, \textcolor{Rhodamine}{\xi_i', \xi_i^*} & \geq 0 \\
-\forall i = 1,...,n \\
 \end{dcases}
 \end{aligned}
 \label{eq:surv_ssvmvb2}
```

Comment:
There is some confusion below (line 227f): blue parts = ranking but gamma = regression. 
Overall, I think I would introduce the $\xi$ for the ranking part $\xi'$, so that the $\xi$ and $xi*$ remain their notation from the regression part and the $\xi'$ is uniquely used for ranking. and use $\mu \sum \xi'$ in ranking, so everything is introduced the same way as it will be used in the hybrid part

Reply:
Line 229: Don't get the meaning of this sentence

Reply:
L255: "wide domain" -> "wide range/space"?

Reply:
> Line 229: Don't get the meaning of this sentence

@adibender  do you remember which?